athina/steps/api.py

```
@@ -35,85 +97,58 @@ class ApiCall(Step):
+                    async with session.request(
+                        method=self.method,
+                        url=self.url,
+                        headers=prepared_headers,
+                        params=prepared_params,
+                        json=json_body,
+                    ) as response:
+                        response_text = await response.text()
+                        return process_response(response.status, response_text)
+
+            except asyncio.TimeoutError:
                 if attempt < retries - 1:
```

Ruff (0.8.2)

Undefined name `retries`

(F821)